model/src/main/java/de/evoila/cf/broker/model/JobProgress.java

```diff
@@ -104,4 +114,36 @@ public String getReferenceId() {
     public void setReferenceId(String referenceId) {
         this.referenceId = referenceId;
     }
+
+    public boolean isInProgress() {
```

Comment:
I think its best practice to compare enums in java with `==` as it might be faster and provide type during compile time

Reply:
The fields are string constants, so no option to use enum equals. But one could change these values to an enum.